test/fetch.test.js

```diff
@@ -3,16 +3,24 @@
 const fs = require('fs');
 
 const test = require('tape');
+const { mockS3Http } = require('../lib/node-pre-gyp.js');
 const nock = require('nock');
 const install = require('../lib/install.js');
 
+test.onFinish(() => {
+  mockS3Http('on');
+});
+
 test('should follow redirects', (t) => {
+  // clear existing mocks
+  const was = mockS3Http('off');
+
   // Dummy tar.gz data, contains a blank directory.
   const targz = 'H4sICPr8u1oCA3kudGFyANPTZ6A5MDAwMDc1VQDTZhAaCGA0hGNobGRqZm5uZmxupGBgaGhiZsKgYMpAB1BaXJJYBHRKYk5pcioedeUZqak5+D2J5CkFhlEwCkbBKBjkAAAyG1ofAAYAAA==';
-
   // Mock an HTTP redirect
-  const n = nock('https://mapbox-node-pre-gyp-public-testing-bucket.s3.us-east-1.amazonaws.com')
-    .get(/\/node-pre-gyp\/node-pre-gyp-test-app1\/v0.1.0\/Release\/node-v\d+-\S+.tar.gz/)
+  const scope = nock('https://mapbox-node-pre-gyp-public-testing-bucket.s3.us-east-1.amazonaws.com')
+    // .get(/\/node-pre-gyp\/node-pre-gyp-test-app1\/v0.1.0\/Release\/node-v\d+-\S+.tar.gz/)
```

Comment:
Is this line supposed to be fully commented or was this a temporary testing change that should be reverted?

Reply:
it was a temp change that should have been reverted. thanks for catching it.